templates/quickstart-crowd-dc.template.yaml

```diff
@@ -1404,7 +1404,57 @@ Resources:
             Principal:
               AWS:
                 - !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:root"
-            Action: 'kms:*'
+            Action:
+            - kms:CancelKeyDeletion
+            - kms:ConnectCustomKeyStore
```

Comment:
I think, for now, we can keep this as it is the same as the previous implementation but I think we have `Actions` in the list that are not necessary for the stack deployment and operation. As this list is just a straight translation of `kms:*` now it is not changing behaviour.

But the other option would be removing policies that are clearly not necessary. E.g. `kms:ConnectCustomKeyStore` and `kms:CreateCustomKeyStore`... I believe these and others are not actually used by the tempaltes.

Reply:
Great, will create a ticket for future ref